src/expiring_market.sol

```diff
@@ -1,15 +1,18 @@
-pragma solidity ^0.4.2;
+pragma solidity ^0.4.8;
 
-import 'simple_market.sol';
+import "./simple_market.sol";
 
 // Simple Market with a market lifetime. When the lifetime has elapsed,
 // offers can only be cancelled (offer and buy will throw).
 
 contract ExpiringMarket is SimpleMarket {
+    uint constant public lifetime = 1 weeks;
```

Comment:
I wanted to be able to deploy it more easily (by just deploying the `.bin` file produced by `solc`), but I guess it's not a big deal to append the constructor argument.